### PR TITLE
djlint --reformat, fix elements nesting

### PR DESCRIPTION
**Pull request by mariuskava**
## Done

applied djlint --reformat and fixed element nesting.

## QA

- Check out this feature branch
- Run the site using the command `dotrun`
- View the site locally in your web browser at: `http://0.0.0.0:8002/careers/application/<token>`
- Nothing should have changed in the view

### CODE REVIEW

File: templates/careers/application/index.html [hunk 1]
```diff
@@ -177,30 +220,40 @@ <h3 class="p-heading--2 p-process-stepped-list__title--stepper-{% if 'stage_prog
...
+                    src="https://youtube.com/embed/ik8AXTjUbS4"
+                    width="560"
+                    height="315"
+                    frameborder="0"
+                    title="How we hire at Canonical"></iframe>
+          </div>
         </div>
-      </div>
-    </section>
-    <section class="p-strip">
```
Comment (mariuskava) [inline templates/careers/application/index.html:193]: this element was spanning outside if block

File: templates/careers/application/partial/_early-stage.html [hunk 2]
```diff
@@ -1,28 +1,37 @@
...
+  <h3 class="p-heading--2 p-process-stepped-list__title--stepper-{% if 'stage_progress' in application and application['stage_progress']['late_stage'] %}complete{% elif 'stage_progress' in application and not application['stage_progress']['late_stage'] and application['stage_progress']['early_stage'] %}active{% else %}not-completed{% endif %}">
+    Early stage
+  </h3>
   <div class="p-stepped-list__content">
-    <p>Our interviews are a blend of technical and competency based interviews. If successful you will also meet a member of our Talent Team.</p>
-    {% if "stage_progress" in application and application["stage_progress"]["late_stage"] %}
-    <a href="" onclick="showProgressDetail('early-stage'); event.preventDefault();" class="show-more-early-stage">Show more</a>
-    <div class="progress-detail-early-stage u-hide">
```
Comment (mariuskava) [inline templates/careers/application/partial/_early-stage.html:7]: this element was spanning outside if block

File: templates/careers/application/partial/_late-stage.html [hunk 3]
```diff
@@ -1,28 +1,37 @@
...
 <li class="p-stepped-list__item">
-  <h3 class="p-heading--2 p-process-stepped-list__title--stepper-{% if 'stage_progress' in application and application['stage_progress']['offer'] %}complete{% elif 'stage_progress' in application and not application['stage_progress']['offer'] and application['stage_progress']['late_stage'] %}active{% else %}not-completed{% endif %}">Late stage</h3>
+  <h3 class="p-heading--2 p-process-stepped-list__title--stepper-{% if 'stage_progress' in application and application['stage_progress']['offer'] %}complete{% elif 'stage_progress' in application and not application['stage_progress']['offer'] and application['stage_progress']['late_stage'] %}active{% else %}not-completed{% endif %}">
+    Late stage
+  </h3>
   <div class="p-stepped-list__content">
-    <p>During this stage you will meet with some of our more senior leads and executives who will form part of your management team.</p>
-    {% if "stage_progress" in application and application["stage_progress"]["offer"] %}
-    <a href="" onclick="showProgressDetail('late-stage'); event.preventDefault();" class="show-more-late-stage">Show more</a>
-    <div class="progress-detail-late-stage u-hide">
```
Comment (mariuskava) [inline templates/careers/application/partial/_late-stage.html:7]: this element was spanning outside if block

**Review by nathanclairmonte — APPROVED** [review]
LGTM! 🚀

**Pull request merged**